examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,202 @@
+        'budget': {
+            'budgetId': budgetId
+        },
+        'networkSetting': {
+            'targetGoogleSearch': 'true',
+            'targetSearchNetwork': 'true',
+        }
+    }
+    campaign_operations = [{
+        'operator': 'ADD',
+        'operand': campaign
+    }]
+    results = campaign_service.mutate(campaign_operations)
+    createdCampaign = results['value'][0]
+    print("CreatedCampign with ID {} and name {} was created". \
+           format(createdCampaign['id'],createdCampaign['name']))
+    return createdCampaign['id']
+
+
+def createAdGroup(client, campaign_id):
+    ad_group_service = client.GetService('AdGroupService', 'v201809')
+    ad_group =   {
+        'name': 'Earth to Mars Cruise #%s' % uuid.uuid4(),
+        'campaignId': campaign_id,
+        'status': 'ENABLED',
+        'biddingStrategyConfiguration' : {
+        'bids': [{
+          # The 'xsi_type' field allows you to specify the xsi:type of the
+          # object being created. It's only necessary when you must provide
+          # an explicit type that the client library can't infer.
+            'xsi_type': 'CpcBid',
+            'bid': {
+                'microAmount': 10000000
+            }
+        }]
+        },
+        'adGroupAdRotationMode': 'OPTIMIZE'
+    }
+
+    adgroup_operations = [{
+        'operator': 'ADD',
+        'operand': ad_group
+    }]
+    results = ad_group_service.mutate(adgroup_operations)
+    createdAdgroup = results['value'][0]
+    print("Ad group with ID {} and name {} was created".\
+        format(createdAdgroup['id'],createdAdgroup['name']))
+    return createdAdgroup['id']
+
+
+def createTextAds(client, adGroupId):
+    ad_group_service = client.GetService('AdGroupAdService', 'v201809')
+    operations = []
+    for i in range(NUMBER_OF_ADS):
+        operation =  {
+            'xsi_type': 'AdGroupAd',
+            'adGroupId': adGroupId,
+            # Additional properties (non-required).
+            'status':  'PAUSED',
+            'ad': {
+                'xsi_type': 'ExpandedTextAd',
+                'headlinePart1': ('Cruise #%s to Mars'
+                            % str(uuid.uuid4())[:8]),
+                'headlinePart2': 'Best Space Cruise Line',
+                'headlinePart3': 'For Your Loved Ones',
+                'description': 'Buy your tickets now!',
+                'description2': 'Discount ends soon',
+                'finalUrls': ['http://www.example.com/']
+            }
+        }
+        adgroup_operations = {
+            'operator': 'ADD',
+            'operand': operation
+         }
+        operations.append(adgroup_operations)
+    results = ad_group_service.mutate(operations)
+    for result in results['value']:
+        print('Expanded text ad with ID {} and headline {}-{} {} was created'\
+              .format(result['ad']['id'],result['ad']['headlinePart1'],\
+               result['ad']['headlinePart2'],result['ad']['headlinePart3']))
```

Comment:
End the first line with `.format(` if it fits under 80 characters, otherwise break the string up, like this:

```python
print(' Expanded test ad ... ' + 
      '... was created'.format(
```

Always put spaces after commas and if needed add new lines after them if the second parameter does beyond 80 chars. This makes the statement easier to read, so for example:

```python
print(' Expanded test ad ... '
      '... was created'.format(result['ad']['id'],
      result['ad']['headlinePart1'], result['ad']['headlinePart2'],
      result['ad']['headlinePart3']))
```